### PR TITLE
Qs Transparency & data usage view on QS footer

### PR DESCRIPTION
**Pull request by ShevT**

### CODE REVIEW

File: res/values/cr_strings.xml [hunk 1]
```diff
@@ -1020,4 +1020,7 @@
     <!-- Battery temperature -->
     <string name="battery_temperature">Battery temperature</string>
 
+    <!-- Qs Transparency -->
+    <string name="qs_transparency_title">Qs Transparency</string>
```
Comment (gwolf2u) [inline res/values/cr_strings.xml:1024]: QS with capital S

File: res/values/cr_strings.xml [hunk 2]
```diff
@@ -1020,4 +1020,7 @@
...
+    <string name="qs_transparency_summary">Choose how transparent you want your qs to be (Requires SystemUI Restart)</string>
```
Comment (gwolf2u) [inline res/values/cr_strings.xml]: QS also
and SystemUI restart is not quite easy to do as we don't have dedicated button
Maybe say System restart ?

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: res/values/cr_strings.xml (hunk 2 above)
Comment (ShevT) [reply res/values/cr_strings.xml]: Ok

Comment (ShevT) [reply res/values/cr_strings.xml]: Done

**Pull request closed without merging**